src/fast++-read_input.cpp

```diff
@@ -1656,6 +1663,11 @@ bool check_input(options_t& opts, input_state_t& state) {
         }
     }
 
+    // If no photometry is present then disable the auto-scaling of the spectrum
+    if (state.flux.dims[1] == 0) {
```

Comment:
```suggestion
    if (state.no_filt.empty()) {
```

Reply:
NB: at this stage ```state.flux.dims[1]``` is no longer zero because is has already been resized to include the spectrum data.